src/blocks/custom_dbus.rs

```diff
@@ -27,12 +28,18 @@ pub struct CustomDBus {
     id: usize,
     text: TextWidget,
     status: Arc<Mutex<CustomDBusStatus>>,
+    timeout: Option<Duration>,
+    clear_pending: Option<Instant>,
 }
 
 #[derive(Deserialize, Debug, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct CustomDBusConfig {
     pub name: String,
+
+    /// Timeout for clearing the block output after an update (in seconds)
+    #[serde(default, deserialize_with = "deserialize_opt_duration")]
+    pub timeout: Option<Duration>,
```

Comment:
Perhaps it could be added as an optional parameter to the DBus API?

Reply:
Sure. Though I think the current API needs some reworking so you can set it without having to supply icon/state as well.

Will leave it for later